lib/tasks/release_note.rake

```diff
@@ -82,11 +82,16 @@ def find_changes(repo, from, to)
   changes = {}
   CHANGE_TYPES.each { |ct| changes[ct] = Set.new }
 
+  repo_path =
+    `git -C #{repo} remote get-url origin`.match(%r{github.com[/:](?<repo>(?:(?!\.git).)*)}).try(
```

Comment:
Retrieving the repo path here makes the code work for both Discourse core and plugins.